### PR TITLE
Add Dockerfile and docker-compose for local

### PR DESCRIPTION
**Pull request by trungthanhtong**
Add render.yaml for render.com

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+services:
+  app:
+    ports:
+      - 5500:80
+    environment:
+      - APACHE_DOCUMENT_ROOT=/var/www/html/public
+    build:
+      context: .
+    volumes:
+      # - ./config/vhost.conf:/etc/apache2/sites-available/000-default.conf
```
Comment (tylerblueink) [inline docker-compose.yml]: Do we want to leave this comment here? Or just remove it?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer \
+    && apt-get update && apt-get install -y git libzip-dev unzip \
+    && docker-php-ext-install zip \
+    && a2enmod rewrite headers 
+
+COPY . /var/www/html
+
+COPY ./config/vhost.conf /etc/apache2/sites-available/000-default.conf
+
+RUN cd /etc/apache2/sites-available/ && a2ensite *
```
Comment (zlove) [inline Dockerfile]: @trungthanhtong - why not just a2ensite 000-default.conf? Could there be anything else in sites-available we might not want to enable?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+version: '3.4'
```
Comment (zlove) [inline docker-compose.yml:1]: @trungthanhtong - can we still run this via docker-compose for local development?

**Review by zlove — COMMENTED** [review]
@trungthanhtong - I left once comment in the Dockerfile re a2ensite. Once that's resolved, I think this looks good to approve and merge.

File: Dockerfile (hunk 2 above)
Comment (trungthanhtong) [reply Dockerfile]: @zlove I modified it so just  a2ensite 000-default.conf

File: docker-compose.yml (hunk 3 above)
Comment (trungthanhtong) [reply docker-compose.yml:1]: @zlove We can still run it via docker-compose

**Review by tylerblueink — APPROVED** [review]

**Review by zlove — APPROVED** [review]

**Pull request merged**